cropper/src/main/java/com/canhub/cropper/PickImageContract.kt

```diff
@@ -0,0 +1,47 @@
+package com.canhub.cropper
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.activity.result.contract.ActivityResultContract
+import com.canhub.cropper.CropImage.getPickImageResultUriContent
+
+/**
+ * An ActivityResultContract to prompt the user to pick an image, receiving
+ * a Uri for that image that allows you to use
+ * android.content.ContentResolver#openInputStream(Uri) to access the raw data.
+ * <p>
+ * Set the boolean input flag to true to include the camera in the options presented to the user.
+ * <p>
+*/
+
+class PickImageContract : ActivityResultContract<Boolean, Uri?>() {
+
+    private var context: Context? = null
+
+    override fun createIntent(context: Context, input: Boolean): Intent {
+
+        this.context = context
+
+        return CropImage.getPickImageChooserIntent(
+            context = context,
+            title = context.getString(R.string.pick_image_intent_chooser_title),
+            includeDocuments = false,
+            includeCamera = input
+        )
+    }
+
+    override fun parseResult(
+        resultCode: Int,
+        intent: Intent?
+    ): Uri? {
+        if (intent != null) {
+            context?.let {
+                context = null
+                return getPickImageResultUriContent(it, intent)
```

Comment:
This method is returning only `UriContent` missing `UriFilePath`. We need to return both.

Sorry I should have spotted this before. 


To give a little more context. With the **Android 10/11** changes `UriContent` doesn't represent a `file path` anymore, using the `content://` schema for it URI. What is right by definition. 
But many users of this library and other ones, got used of using URI as file path for saving the crop image.
So we split `UriContent` and `UriFilePath` as user request. In previous versions of Android `UriContent` can be in the file schema, in recent ones in content schema. 
So now we have `UriFilePath` that always represent a file schema.

Why not put all in this `file` schema? To save space/memory, since some users doesn't need it.

Reply:
So what do we do? Have it return a custom class with two uris?

Reply:
Is better that we avoid the filePath generation, if the user will not use it. 
But would be nice to make the method available for usage, so we generate only if the user need.

No idea the best way of doing this =/

Reply:
We could make `PickImageContract` an open class so users can override the getResult method and parse the result differently

Reply:
Nice!